.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: run-tests
+
+on:
+  workflow_call:
+    outputs:
+      pr_number:
+        description: "The PR number"
+        value: ${{ jobs.test.outputs.pr_number }}
```

Comment:
If we want these tests to run before for Dependabot PRs this workflow needs to be chained to the merge dependabot one.  
Can add this job and remove the `automerge-dependabot.yml` workflow
```
  call-dependabot-pr-workflow:
    needs: test
    if: ${{ success() && github.actor == 'dependabot[bot]' }}
    uses: cloudfoundry/cloud-service-broker/.github/workflows/dependabot-test.yml@main
    with:
      pr_number: ${{ github.event.number }}
```

Reply:
I actually did this and commited